proc-macros/src/render_server.rs

```diff
@@ -272,11 +273,23 @@ impl RpcDescription {
 			let decode_fields = params.iter().map(|(name, ty)| {
 				if is_option(ty) {
 					quote! {
-						let #name: #ty = seq.optional_next()?;
+						let #name: #ty = match seq.optional_next() {
+							Ok(v) => v,
+							Err(e) => {
+								log::error!("Error parsing optional {:?} as {:?}: {:?}", stringify!(#name), stringify!(#ty), e);
```

Comment:
You can concatenate string literals at compile time with macros:

```suggestion
								log::error!(concat!("Error parsing optional \"", stringify!(#name), "\" as \"", stringify!(#ty), "\": {:?}"), e);
```